### DISCUSSION TITLE
SDXL working but output quality is very poor

### DISCUSSION THREAD

**Discussion by pushstartvr** [continue-revolution/sd-webui-animatediff#307] [General]
Hello, unsure where to post so I just came here.  as the title says.
Single image generation is great compared to motion module generation, just like v15 for 512x512, however the output for SDXL is much worse
SDXL video:
https://github.com/continue-revolution/sd-webui-animatediff/assets/25425449/c00bf120-8836-4a12-a90a-9565e8cfea2d
SDXL: single image
![36727-2414275260-robot war, (gigantic robot walkers), landscape, layered cloud atmosphere, sci-fi, digital painting, rich color, smooth, Unreal E](https://github.com/continue-revolution/sd-webui-animatediff/assets/25425449/6e92311f-b1f9-4cbe-a474-03f198462cf5)

its amazing though, centre frame subjects at high res, cat wait til I can get it working properly

**Comment by continue-revolution** [comment]
Unfortunate, just as what researchers from Shanghai AI Lab have named their model - “beta”, it is not very stable in terms of performance. A good news is that they will release more models in the future, which I expect to be more robust.

**Reply by pushstartvr** [reply]
I'm no expert though so maybe I am doing something wrong.  I've only been on animatediff for a week or so on Auto1111 webUI, haven't attempted comfyUI at all
